airmozilla/main/templates/main/event.html

```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
+              <a class="share" href="https://www.facebook.com/sharer/sharer.php?u={{ make_absolute(url('main:event', event.slug)) }}" target="_blank">Share on Facebook</a>
```

Comment:
From here https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#url any special character should be encoded